README.md

```diff
@@ -123,9 +143,9 @@ environment variable `NODE_ENV=production`. These differences include:
 
 Heroku relies upon the [habitat](https://github.com/brianloveswords/habitat) environment variables loader library.  The config for this can be found in [lib/environments/heroku.js](lib/environments/heroku.js).
 
-When configuring your environment variables on heroku, they should be prefixed with the string that is passed to the habitat constructor. For example, if we pass the string "openbadges" to our habitat constructor, like so: `new Habitat("openbadges")`, then in heroku our protocol env var would be "OPENBADGES_PROTOCOL" (with, in this case, a value set to either http or https).
+When configuring your environment variables on Heroku, they should be prefixed with the string that is passed to the habitat constructor. For example, if we pass the string "open badges" to our habitat constructor, like so: `new Habitat("open badges")`, then in Heroku our protocol env var would be "OPENBADGES_PROTOCOL" (with, in this case, a value set to either HTTP or https).
```

Comment:
The string `openbadges` is acceptable and correct.

Also, if we uppercase HTTP we should probably uppercase HTTPS too.